src/transform/ardtransform.jl

```diff
@@ -23,7 +23,10 @@ Create an [`ARDTransform`](@ref) with vector `fill(s, dims)`.
 """
 ARDTransform(s::Real, dims::Integer) = ARDTransform(fill(s, dims))
 
-@functor ARDTransform
+function ParameterHandling.flatten(::Type{T}, t::ARDTransform{S}) where {T<:Real,S}
+    unflatten_to_ardtransform(v::Vector{T}) = ARDTransform(convert(S, map(exp, v)))
+    return convert(Vector{T}, map(log, t.v)), unflatten_to_ardtransform
```

Comment:
Could this be written as
```suggestion
    return map(T ∘ log, t.v), unflatten_to_ardtransform
```
?

Reply:
IIRC ParameterHandling wants a `Vector`. Might be still useful though to fuse the `T` and `log` and only use a `convert(Vector, ...)`.

Reply:
```suggestion
    return convert(Vector, map(T ∘ log, t.v)), unflatten_to_ardtransform
```